packages/algjulia-service/src/decapodes-service/plotting.jl

```diff
@@ -9,3 +9,20 @@ function Base.reshape(::Heatmap, data)
     l = floor(Int64, sqrt(length(data)))
     reshape(data, l, l)
 end
+
+# TODO size fixed
```

Comment:
Commented code should be fixed and uncommented, or removed. I have no preference about which. If we're not producing heat maps in Julia, we could just delete this stuff.

Reply:
IIRC we rely on Julia to format the data for plotting, so I can see this design being useful if we want to specify different ways of plotting which would require different output formats. As of now it's unused and will be deleted.